src/common/Screen.js

```diff
@@ -116,6 +120,7 @@ class Screen extends PureComponent<Props> {
           <ModalNavBar canGoBack={canGoBack} title={title} />
         )}
         <OfflineNotice />
+        {shouldShowLoadingBanner === true && <LoadingBanner />}
```

Comment:
nit: simplify to drop the `=== true` -- it's already boolean :slightly_smiling_face:

Reply:
Hmm, but it's an optional property of Props (`shouldShowLoadingBanner?: boolean`), so it could be `undefined`.

Flow says this:

```
Sketchy null check on boolean [1] which is potentially false. Perhaps you meant to check for null or undefined [1]?
(sketchy-null-bool)
```

Reply:
Hmm, that's odd -- that warning should be specific to a type like `boolean | void`, not `boolean`...

Ah, I see. It's because the type is written as optional in `Props`. Instead, when a value is supplied in `defaultProps`, that prop should be non-optional in `Props`.

IOW -- well, the context is that there's really two versions of what the type of "the props of this component" are:
* the version that applies to callers;
* the version that applies to the implementation, after `defaultProps` has had its say.

And then another way to say the above rule is that `Props` (or more exactly, the type argument to `Component` or `PureComponent`, which we conventionally call `Props`) describes the latter, inward-facing version of "the props of this component". Flow then automatically looks at `defaultProps` and uses it to figure out the former, outward-facing version. Here's the relevant bit of the Flow docs:
https://flow.org/en/docs/react/components/#toc-using-default-props

Then Flow should see that this `shouldShowLoadingBanner` local variable is indeed already boolean :slightly_smiling_face:

Reply:
Yup! Quite handy.

I've also just pushed a version of this comment as f3694dce2.

Reply:
Great, thanks!